packages/devtools_app/integration_test/test_infra/run/_test_app_driver.dart

```diff
@@ -51,6 +53,9 @@ abstract class _TestApp with IOMixin {
   /// of app.
   final String testAppPath;
 
+  /// The device the test app should run on, e.g. flutter-tester, chrome.
+  final String testAppDevice;
```

Comment:
let's use the enum value here (see previous comment)

Reply:
Done!